src/app.js

```diff
@@ -5,14 +5,26 @@ var app = express();
 
 app.set('port', (process.env.PORT || 3000));
 
+app.use(express.static('public'));
+
 require('./database');
 require('./seed');
 
-app.use('/',express.static('public'));
 app.use(parser.json());
-
 app.use('/api', router);
 
+app.get('/', function(req, res){
+  res.sendFile('/public/index.html',{root:__dirname + '/..'});
+});
+
+app.get('/event*', function(req, res){
+  res.sendFile('/public/event.html',{root:__dirname + '/..'});
+});
+
+app.get('/admin*', function(req, res){
+  res.sendFile('/public/admin.html',{root:__dirname + '/..'});
+});
+
```

Comment:
Curt, could you revert the changes in this file? The remove line 11 covers everything you added.